src/todo/items/todo.ts

```diff
@@ -268,7 +293,10 @@ class Todo extends Item {
           startIndex = match ? match[0].indexOf ( match[1] ) : ( firstChar ? firstChar.index : this.lineNextText.length ),
           endIndex = match ? match[0].length : startIndex;
 
-    this.lineNextText = `${this.lineNextText.substring ( 0, startIndex )}${symbol ? `${symbol} ` : ''}${this.lineNextText.substring ( endIndex )}`;
```

Comment:
This function should also be not touched, it just replaces the current symbol with the next one, I don't see why it should be changed.